data/news/contra_costa.json

```diff
@@ -3,12 +3,6 @@
   "title": "Contra Costa County COVID-19 News",
   "home_page_url": "https://www.coronavirus.cchealth.org/health-services-updates",
   "items": [
-    {
-      "id": "https://www.coronavirus.cchealth.org/health-orders",
-      "url": "https://www.coronavirus.cchealth.org/health-orders",
-      "title": "Health Order: Updated Mass Quarantine Order",
-      "date_published": "2020-10-08T00:00:00-07:00"
-    },
```

Comment:
This news item hasn’t actually been removed. It looks like the format of news items has changed a bit and some are now getting skipped by our parsing approach. See https://github.com/sfbrigade/data-covid19-sfbayarea/issues/142